account_import_line_multicurrency_extension/models/account_statement_from_invoice.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Vincent Renaville (Camptocamp)
+#    Copyright 2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, api
+
+
+class account_statement_from_invoice_lines(models.TransientModel):
+    """
+    Generate Entries by Statement from Invoices
+    """
+    _inherit = "account.statement.from.invoice.lines"
+
+    @api.multi
+    def populate_statement(self):
```

Comment:
It will be possible to improve it if https://github.com/odoo/odoo/pull/7608 is merged